python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -532,6 +537,49 @@ def resolve_title(self, _graphene_info: ResolveInfo) -> Optional[str]:
     def resolve_description(self, _graphene_info: ResolveInfo) -> Optional[str]:
         return self._backfill_job.description
 
+    def resolve_logEvents(self, graphene_info: ResolveInfo, cursor: Optional[str] = None):
+        from ..schema.instigation import (
+            GrapheneInstigationEvent,
+            GrapheneInstigationEventConnection,
+        )
+        from ..schema.logs.log_level import GrapheneLogLevel
+
+        backfill_id = self._backfill_job.backfill_id
+        # TODO find a better way to keep the log key prefix in sync. maybe store as part of the asset backfill data?
```

Comment:
resolve this TODO

Reply:
don't think you need to store it - can have a method on `PartitionBackfill` that returns the value you are making below. What are you worried about syncing with?

Reply:
mostly just if we changed the prefix at some point and needed backcompat for logs with the old prefix. i like the method on `PartitionBackfill`